bridges/relays/messages/src/message_race_delivery.rs

```diff
@@ -770,9 +750,6 @@ mod tests {
 			max_messages_weight_in_single_batch: Weight::from_parts(4, 0),
 			max_messages_size_in_single_batch: 4,
 			latest_confirmed_nonces_at_source: vec![(header_id(1), 19)].into_iter().collect(),
-			lane_source_client: TestSourceClient::default(),
-			lane_target_client: TestTargetClient::default(),
-			metrics_msg: None,
```

Comment:
Sorry, one last thing. Just checking, but I guess that these fields were not used anywhere ?

Reply:
Yep, clippy wanted to see them gone. They were apparently never read, only written to.

Reply:
Ok, thanks ! Looks good then !